setup.py

```diff
@@ -19,6 +19,6 @@
     include_package_data=True,
     license=package['license'],
     description=package['description'] if 'description' in package else package_name,
-    install_requires=['dash', 'dash-html-components', 'scikit-image', 'Pillow',
+    install_requires=['dash>=1.6.0', 'dash-html-components', 'scikit-image', 'Pillow',
```

Comment:
now needs a version of dash with async support

Reply:
Can we take html, dcc, and plotly out of this list since they all come in with dash?

Reply:
https://github.com/plotly/dash-canvas/pull/29/commits/2b073cbecb845196b240db21beea159801e4a588